modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -1973,18 +5544,18 @@ NOTE: This input source does not support having multiple event notifications on
 [[reconnect]]
 === Reconnect
```

Comment:
Did you update the Connect and Reconnect Forever blocks per the instructions for editing references? There is a 'blocking' field in the Reconnect Forever block.